samples/app/MeshAssimp.cpp

```diff
@@ -1106,4 +1106,12 @@ void MeshAssimp::processGLTFMaterial(const aiScene* scene, const aiMaterial* mat
         sRGBColorA baseColorFactorCast = *reinterpret_cast<sRGBColorA*>(&baseColorFactor);
         outMaterials[materialName]->setParameter("baseColorFactor", baseColorFactorCast);
     }
+
+    aiBool isSpecularGlossiness = false;
+    if (material->Get(AI_MATKEY_GLTF_PBRSPECULARGLOSSINESS, isSpecularGlossiness)
```

Comment:
Breaking `if` into two lines looks a little strange, you can make it a single line and still be under the 100 column count